OrbitQt/topdownwidget.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_TOP_DOWN_WIDGET_H_
+#define ORBIT_QT_TOP_DOWN_WIDGET_H_
+
+#include <QSortFilterProxyModel>
+#include <memory>
+
+#include "TopDownView.h"
+#include "ui_topdownwidget.h"
+
+class TopDownWidget : public QWidget {
+  Q_OBJECT
+
+ public:
+  explicit TopDownWidget(QWidget* parent = nullptr)
+      : QWidget{parent}, ui_{std::make_unique<Ui::TopDownWidget>()} {
+    ui_->setupUi(this);
+    connect(ui_->topDownTreeView,
```

Comment:
Qt's new signal-slot syntax is preferable: https://wiki.qt.io/New_Signal_Slot_Syntax

Reply:
Done, thank you.